packages/sentinel/src/api/index.ts

```diff
@@ -55,11 +57,14 @@ export class SentinelClient extends BaseApiClient {
 
   public async update(sentinelId: string, sentinel: UpdateSentinelRequest): Promise<CreateSentinelResponse> {
     const currentSentinel = await this.get(sentinelId);
+
+    const updatedSentinel = await this.constructSentinelRequest({
+      ...this.toCreateSentinelRequest(currentSentinel),
```

Comment:
we should move the `toCreateSentinelRequest()` call to the get method as @spalladino suggested, so users are working with this type by default

``` TS
  public async get(sentinelId: string): Promise<CreateSentinelRequest> {
    return this.apiCall(async (api) => {
      return this.toCreateSentinelRequest(await api.get(`/subscribers/${sentinelId}`));
    });
  }
```

Reply:
Isn't that breaking backwards compatibility?

I decided to shorten the scope of this PR to solve our client's need first because the `list` endpoint also has to be changed to use the `toCreateSentinelRequest()`. But if @spalladino think it's worth to change the shape, I can proceed to update both endpoints and the corresponding docs

Reply:
Opened an issue for this, I think it requires further exploration and not just rolling out

https://github.com/OpenZeppelin/defender/issues/2547